lib/gauche/interactive/editable-reader.scm

```diff
@@ -57,7 +57,7 @@
   (if-let1 console (make-default-console :if-not-available #f)
     (let ([ctx (make <line-edit-context>
                  :console console
-                 :prompt (^[] (putstr console (get-prompt-string)))
+                 :prompt (^[] (display (get-prompt-string)))
```

Comment:
Good catch.  I guess I changed the design of line-edit.scm but forgot to update this.